rpmlb/cli.py

```diff
@@ -75,6 +75,11 @@
     help='Resume build from specified position.',
 )
 @click.option(
+    '--retry', metavar='N',
+    type=click.IntRange(min=0), default=0,  # no negatives
+    help='Retry failed build N times.',
```

Comment:
I want you to include the word "Retry number" in the help message.
And `N` can be changed to another metaver name.